DNN Platform/Library/Entities/Modules/Settings/SettingsRepository.cs

```diff
@@ -63,23 +63,16 @@ public void SaveSettings(ModuleInfo moduleContext, T settings)
 
                         if(string.IsNullOrEmpty(settingValueAsString))
                         {
-                            if (settingValue is DateTime)
-                            {
-                                settingValueAsString = ((DateTime)settingValue).ToString("u");
-                            }
-                            else
-                            {
-                                settingValueAsString = settingValue.ToString();
-                            }
+                            settingValueAsString = GetSettingValueAsString(settingValue);
```

Comment:
Nono. It's about the app starting up in a different locale. You can't imagine how unsure you can be of the thread locale in multilingual sites. It really depends on who's logged in, etc.